src/html/recipe.html

```diff
@@ -1,18 +1,19 @@
-<!doctype html>
-
-<head>
+<!DOCTYPE html>
+<html>
+  <head>
     <title>RecipeCleaner</title>
     <meta charset="utf-8">
-
     <link rel="stylesheet" href="/css/recipe.css">
-    <link rel="stylesheet" href="/css/options.css">
-
-    <script src="/js/vendor.js"></script>
-    <script src="/js/options.js"></script>
-    <script src="/js/recipe.js"></script>
-</head>
-
-<body>
-    <div id="options"></div>
```

Comment:
Was removing the options gear intentional?

Reply:
I believe it was, but I can't remember why at this point I removed it. It might be because the gear would be broken when the HTML was saved.

Reply:
I think I was trying to take the straightest line toward saving recipes in a readable format that also included the recipe image, and could still be post-processed, and / or searched directly.

I dunno about you, but I always find that printing web pages is a bit frustrating, but it is worth investigating.